src/Evolution/Systems/CurvedScalarWave/BoundaryConditions/BoundaryCondition.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pup.h>
+
+#include "Domain/BoundaryConditions/BoundaryCondition.hpp"
+#include "Domain/BoundaryConditions/Periodic.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace CurvedScalarWave::BoundaryConditions {
+
+template <size_t Dim>
```

Comment:
Can you place `/// \cond` and `/// \endcond` around the forward decls please? Doxygen sometimes picks up the empty documentation and has it hide the one we actually want :(